src/helpers/validation.js

```diff
@@ -0,0 +1,8 @@
+const validationString = (text) => {       
+  return (typeof text !== 'string'|| text === "");
```

Comment:
change the conditions, otherwise it’s not logical. You return true when you have a non-string type and an empty field